packages/bot/src/stores/journal-store.js

```diff
@@ -25,34 +17,25 @@ export default class JournalStore {
 
     @action.bound
     onLogSuccess(data) {
-        this.pushMessage(data, message_types.success);
+        this.pushMessage(data, message_types.SUCCESS);
     }
 
     @action.bound
     onError(data) {
-        if (unrecoverable_errors.includes(data.name)) {
-            this.root_store.contract_card.clear();
-            this.root_store.run_panel.is_continue_trading = false;
-        } else {
-            this.root_store.run_panel.is_continue_trading = true;
-        }
-        this.root_store.run_panel.setActiveTabIndex(2);
-        this.root_store.run_panel.is_error_happened = true;
-        this.pushMessage(data , message_types.error);
+        this.pushMessage(data , message_types.ERROR);
     }
 
     @action.bound
     onNotify(data) {
-        this.pushMessage(data , message_types.notify);
-    }
-
-    @action.bound
-    clear() {
-        this.messages = this.messages.slice(0, 0);  // force array update
+        this.pushMessage(data , message_types.NOTIFY);
     }
     
     @action.bound
     pushMessage(data, message_type) {
+        if (!this.root_store.core.client.is_virtual) {
```

Comment:
Won't this disable messages for real money accounts? Should it be `!this.root_store.core.client.is_logged_in`?

Reply:
@aaron-binary Actually I don't think we need it, cause we must print all the messages as long as we are listening to them.